boot/src/main/java/org/netbeans/html/boot/impl/JavaScriptProcesor.java

```diff
@@ -234,18 +235,127 @@ private boolean verifyResourceAtPath(final String res, StandardLocation location
     public Iterable<? extends Completion> getCompletions(Element e,
         AnnotationMirror annotation, ExecutableElement member, String userText
     ) {
+        if (e.getKind() == ElementKind.METHOD && member.getSimpleName().contentEquals("args")) { // NOI18N
+            return argsCompletion(e);
+        }
+        if (e.getKind() == ElementKind.METHOD && member.getSimpleName().contentEquals("body")) { // NOI18N
+            return bodyCompletion(e, userText);
+        }
+        return null;
```

Comment:
Better to return an empty iterable than `null` as suggested by https://docs.oracle.com/javase/7/docs/api/javax/annotation/processing/Processor.html#getCompletions(javax.lang.model.element.Element,%20javax.lang.model.element.AnnotationMirror,%20javax.lang.model.element.ExecutableElement,%20java.lang.String)